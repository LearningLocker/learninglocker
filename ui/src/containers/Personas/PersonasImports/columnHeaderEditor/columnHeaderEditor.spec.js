import React from 'react';
import { shallow } from 'enzyme';
import 'jest-enzyme';
import { fromJS } from 'immutable';
import {
  ColumnHeaderEditorComponent
} from './index';


describe('ColumnHeaderEditorComponent', () => {
  it('should render', () => {
    const rendered = shallow(<ColumnHeaderEditorComponent
      csvHeaders={fromJS(['test1', 'test2', 'test3'])}
      structure={(fromJS({
        test1: {
          columnName: 'test1'
        },
        test2: {
          columnName: 'test2'
        }
      }))} />);

    expect(rendered).toMatchSnapshot();
  });
});
