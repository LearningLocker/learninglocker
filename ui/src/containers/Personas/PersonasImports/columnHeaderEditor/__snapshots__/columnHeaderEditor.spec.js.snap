// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColumnHeaderEditorComponent should render 1`] = `
<WithStyles(withHandlers(render))>
  <ThemedCard
    className="card"
    composeTheme="deeply"
    key="test1"
  >
    <Connect(withHandlers(HeaderItemComponent))
      columnName="test1"
      columnStructure={
        Object {
          "columnName": "test1",
        }
      }
      structure={
        Object {
          "test1": Object {
            "columnName": "test1",
          },
          "test2": Object {
            "columnName": "test2",
          },
        }
      }
    />
  </ThemedCard>
  <ThemedCard
    className="card"
    composeTheme="deeply"
    key="test2"
  >
    <Connect(withHandlers(HeaderItemComponent))
      columnName="test2"
      columnStructure={
        Object {
          "columnName": "test2",
        }
      }
      structure={
        Object {
          "test1": Object {
            "columnName": "test1",
          },
          "test2": Object {
            "columnName": "test2",
          },
        }
      }
    />
  </ThemedCard>
</WithStyles(withHandlers(render))>
`;
